engine/src/test/scala/cromwell/SubFunctionWorkflowSpec.scala

```diff
@@ -16,10 +16,12 @@ class SubFunctionWorkflowSpec extends CromwellTestkitSpec {
         |  String myBamString = "myfilename.bam"
         |  File myBamFile
         |  String swappedStr = sub(myBamString, ".bam$", ".txt")
-        |  String swappedFile = sub(myBamFile, ".bam$", ".txt")
+        |  # at this point myBamFile is not localized so path must be removed
+        |  String swappedFile = sub(sub(myBamFile,"/.*/",""), ".bam$", ".txt")
```

Comment:
yeah, it's not materially different except for the coercion, but also wanted to test out the exact use case that GOTC is using